contrib/opencensus-ext-azure/opencensus/ext/azure/version.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__version__ = '0.1.dev0'
```

Comment:
@c24t I've moved the version.py from `./ext/` to `./ext/azure/` as I need to consume the version info in the exporter code.
I can move version.py for all the other extensions (which gives better consistency), let me know if you want me to do it.